src/pages/account/update-account.tsx

```diff
@@ -0,0 +1,248 @@
+                    beacon owner (if this is a different person or organisation)
+                    later.
+                  </GovUKBody>
+                </FormFieldset>
+
+                <SectionHeading>About you</SectionHeading>
+                <FullName
+                  value={form.fields.fullName.value}
+                  errorMessages={form.fields.fullName.errorMessages}
+                />
+
+                <TelephoneNumber
+                  value={form.fields.telephoneNumber.value}
+                  errorMessages={form.fields.telephoneNumber.errorMessages}
+                />
+                <AccountHolderAddress form={form} />
+              </FormGroup>
+              <Button buttonText="Save these account details" />
+              &nbsp;
+              <Button buttonText="Cancel" />
+            </Form>
+            <IfYouNeedHelp />
+          </>
+        }
+      />
+    </Layout>
+  );
+};
+
+const FullName: FunctionComponent<FormInputProps> = ({
+  value = "",
+  errorMessages,
+}: FormInputProps): JSX.Element => (
+  <FormGroup errorMessages={errorMessages}>
+    <Input id="fullName" label="Your full name" defaultValue={value} />
+  </FormGroup>
+);
+
+const TelephoneNumber: FunctionComponent<FormInputProps> = ({
+  value = "",
+  errorMessages,
+}: FormInputProps): JSX.Element => (
+  <FormGroup errorMessages={errorMessages}>
+    <Input
+      id="telephoneNumber"
+      label="Telephone number"
+      hintText="Search and rescue may use this to try you in an emergency. This can be a mobile or landline. For international numbers include the country code."
+      defaultValue={value}
+    />
+  </FormGroup>
+);
+
+const AccountHolderAddress: FunctionComponent<{ form: FormJSON }> = ({
+  form,
+}: {
+  form: FormJSON;
+}): JSX.Element => (
+  <FormGroup>
+    <FormGroup errorMessages={form.fields.addressLine1.errorMessages}>
+      <Input
+        id="addressLine1"
+        label="Building number and street"
+        defaultValue={form.fields.addressLine1.value}
+      />
+    </FormGroup>
+    <FormGroup>
+      <Input id="addressLine2" defaultValue={form.fields.addressLine2.value} />
+    </FormGroup>
+    <FormGroup errorMessages={form.fields.townOrCity.errorMessages}>
+      <Input
+        id="townOrCity"
+        label="Town or city"
+        defaultValue={form.fields.townOrCity.value}
+      />
+    </FormGroup>
+    <FormGroup errorMessages={form.fields.county.errorMessages}>
+      <Input
+        id="county"
+        label="County (optional)"
+        defaultValue={form.fields.county.value}
+      />
+    </FormGroup>
+    <FormGroup errorMessages={form.fields.postcode.errorMessages}>
+      <Input
+        id="postcode"
+        label="Postcode"
+        defaultValue={form.fields.postcode.value}
+      />
+    </FormGroup>
+  </FormGroup>
+);
+
+function userDidSubmitForm(context: BeaconsGetServerSidePropsContext) {
+  return context.req.method === "POST";
+}
```

Comment:
```suggestion
const userDidSubmitForm = (context: BeaconsGetServerSidePropsContext): boolean => {
  return context.req.method === "POST";
}
```

For consistency as we use lambda fat arrow functions everywhere else.  Plus return type.